bitcoin/CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# Unreleased (changelog done for PRs merged as of June 24)
+
+- Bump MSRV to Rust 1.48.0 [#1729](https://github.com/rust-bitcoin/rust-bitcoin/pull/1729)
+- Add new example code for signature verification[#1776](https://github.com/rust-bitcoin/rust-bitcoin/pull/1776)
```

Comment:
Nit here and for the remainder of the entries: need a space between the item and the PR link. The first item has it, but not the rest.

- So this is what we want [#123](https://github.com/rust-bitcoin/rust-bitcoin/pull/123)
- And not this[#456](https://github.com/rust-bitcoin/rust-bitcoin/pull/456)

Reply:
Oh good catch, thanks, will fix.